spec/models/validations/soft_validations_spec.rb

```diff
@@ -409,13 +409,25 @@
        {
          period: { label: "every 2 weeks", value: 2 },
          scharge: 69,
+       },
+       {
+         period: { label: "weekly", value: 1 },
+         scharge: 801,
+       },
+       {
+         period: { label: "monthly", value: 4 },
+         scharge: 3471,
+       },
+       {
+         period: { label: "every 2 weeks", value: 2 },
+         scharge: 1601,
        }].each do |test_case|
-        it "returns false if scharge is under soft max for general needs #{test_case[:period][:label]}(35)" do
+        it "returns false if scharge is under soft max (35) or over hard max (800) for general needs #{test_case[:period][:label]}" do
```

Comment:
Can these put whether it's a below soft max or over hard max case into the test name instead of the or?

Reply:
Sure!